src/main/java/uk/gov/register/presentation/view/ViewFactory.java

```diff
@@ -55,7 +59,7 @@ public ThymeleafView thymeleafView(String templateName) {
     }
 
     public HomePageView homePageView() {
```

Comment:
shouldn't we pass in the data to this method, rather than having the ViewFactory pass the DAO into the View?